packages/web/src/scss/foundation/_tokens.scss

```diff
@@ -20,6 +20,6 @@ $tokens: (
     --spirit-border-width-200: #{tokens.$border-width-200};
 
     @each $key, $value in $tokens {
-        @include tokens-tools.generate-css-variables($map: $value, $prefix: '#{tokens.$token-prefix}#{$key}');
+        @include tokens-tools.generate-css-variables($map: $value, $prefix: 'spirit-#{$key}');
```

Comment:
Is it right?

Reply:
it is, until we implement css variables prefixes in React & Twig packages we cannot allow user to modify this. 

If they would change the css-variables-prefix token for example to `seduo` then on their :root element they will have tokens generated as `--seduo-spacing-300: 12px; --seduo-spacing...`, but in React they will use <Flex spacingX="space-400" /> the `style="--flex-spacingx: var(--spirit-space-400)` will be used by the component, so it will not work. 

When we make the React (and Twig) package read the prefix from tokens and actually use it, we can switch this to the token too.

Reply:
ok, thanks for the explanation